index.html

```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
+                     <div class="action">
+                        <button>
+                            <span>
+                                See Project
+                            </span>
+                        </button>
+                     </div>
+                </div>
+            </div>
+            <div class="card-works">
+                <div class="snapshoot-portfolio bg-4">
+                </div>
+                <div class="left-block">
+                    <div>
+                        <h1 class="primary-txt">
+                            Multi-Post Stories
+                        </h1>
+                        <div class="frame">
+                            <div class="client">
+                                <h4 class="frame-client-text">CANOPY</h4>
+                            </div>
+                            <ul class="frame-client-detail">
+                                <li>
+                                    <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                        <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                    </svg>
+                                    Back End Dev
+                                </li>
+                                <li>
+                                    <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                        <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                    </svg>
+                                    2015
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <p class="detail-txt">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="tags">
+                        <div>
+                            <p><span class="tag">html</span>
+
+                            </p>
+                        </div>
+                        <div>
+                            <p>
+                                <span class="tag">css</span>
+                            </p>
+                        </div>
+                        <div>
+                            <p>
+                                <span class="tag">javascript</span>
+                            </p>
+                        </div>
+                    </div>
+                     <div class="action">
+                        <button>
+                            <span>
+                                See Project
+                            </span>
+                        </button>
+                     </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-myself">
+
+        </section>
+        <section class="contact">
+        </section>
```

Comment:
- [ ] it is a requirement for this project to include the contact section. Kindly complete this section.

Reply:
Hi @isaacpitwa  🙂 ,
I hope you are doing well,

In this project, we are only asked to build the middle section of the mobile website. Hence, the **contact** section was not included in my last submission. Here's a link to the **[project-requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)**. However, thanks for the great feedback and for your time and cooperation.